paymentsheet/src/main/java/com/stripe/android/paymentsheet/specifications/Spec.kt

```diff
@@ -66,7 +66,8 @@ sealed class SectionFieldSpec(val identifier: IdentifierSpec) {
 
     object Email : SectionFieldSpec(IdentifierSpec("email"))
 
-    object Country : SectionFieldSpec(IdentifierSpec("country"))
```

Comment:
to avoid creating a new `Country` object in order to retrieve its `IdentifierSpec`, we can probably change `IdentifierSpec` to a sealed class and define the corresponding specs as `object`s
```
sealed class IdentifierSpec(val value: String) {
    object Name: IdentifierSpec("name")
    object Email: IdentifierSpec("email")
    object Country: IdentifierSpec("country")
    ...
}

...
data class Country(val onlyShowCountryCodes: Set<String> = emptySet()) :
        SectionFieldSpec(IdentifierSpec.Country)
```

Reply:
see above